### PR TITLE
Feature/add detailed instructions page

### PR DESCRIPTION
**Pull request by GPretadoR**

### CODE REVIEW

**Comment by GPretadoR** [comment]
Please first review PR #3 then I will merge it and then review this PR #4

File: RakutenShop/Scenes/DetailedInstructionsView/DetailedInstructionsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        super.viewDidLoad()
+        stepDescriptionTableView.dataSource = self
+        // Do any additional setup after loading the view.
+    }
+    
+    override func setupViewModel() {
+        guard let viewModel = viewModel else { return }
+        reactive.makeBindingTarget { [unowned self] (weakself, object) in
+            if let number = object.number {
+                self.title = R.string.localizable.recipeDetailInstructionsStepCountText("\(number)")
```
Comment (Haykmnatsakanyan) [inline RakutenShop/Scenes/DetailedInstructionsView/DetailedInstructionsViewController.swift:31]: self -> weakself

File: RakutenShop/Scenes/DetailedInstructionsView/DetailedInstructionsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    }
+    
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return section == 0 ? R.string.localizable.detailedStepesTableViewHeaderIngredientsText() : R.string.localizable.detailedStepesTableViewHeaderEquipmentsText()
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = UITableViewCell(style: .subtitle, reuseIdentifier: R.reuseIdentifier.instructionsTableViewCell.identifier)
+        
+        if indexPath.section == 0 {
```
Comment (Haykmnatsakanyan) [inline RakutenShop/Scenes/DetailedInstructionsView/DetailedInstructionsViewController.swift:61]: I will suggest to use switch, more preferable in this kind of cases.

File: RakutenShop/Scenes/DetailedInstructionsView/DetailedInstructionsViewViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    case ingredient
+    case equipment
+}
+
+class DetailedInstructionsViewViewModel: BaseViewModel {
+    
+    private let context: Context
+    weak var coordinatorDelegate: DetailedInstructionsViewCoordinatorDelegate?
+    
+    let imagesBaseUrl = "https://spoonacular.com/cdn/"
```
Comment (Haykmnatsakanyan) [inline RakutenShop/Scenes/DetailedInstructionsView/DetailedInstructionsViewViewModel.swift:25]: you told me that should pass this value through coordinators using context

File: RakutenShop/Scenes/DetailedInstructionsView/DetailedInstructionsViewViewModel.swift [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    
+    var steps = MutableProperty<Steps>(Steps(number: 0, step: "", ingredients: [], equipment: []))
```
Comment (Haykmnatsakanyan) [inline RakutenShop/Scenes/DetailedInstructionsView/DetailedInstructionsViewViewModel.swift]: I will suggest to change this with 
var steps = MutableProperty<Steps?>(nil)

File: RakutenShop/Scenes/DetailedInstructionsView/DetailedInstructionsViewViewModel.swift (hunk 3 above)
Comment (GPretadoR) [reply RakutenShop/Scenes/DetailedInstructionsView/DetailedInstructionsViewViewModel.swift:25]: This one is different :)

**Pull request merged**